oshi-demo/src/main/java/oshi/demo/jmx/JMXOshiAgentServer.java

```diff
@@ -0,0 +1,338 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2022 The OSHI Project Contributors: https://github.com/oshi/oshi/graphs/contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package oshi.demo.jmx;
+
+import com.sun.jna.Platform;
+import oshi.PlatformEnum;
+import oshi.SystemInfo;
+import oshi.demo.jmx.api.JMXOshiAgent;
+import oshi.demo.jmx.api.StrategyRegistrationPlatformMBeans;
+import oshi.demo.jmx.strategiesplatform.WindowsStrategyRegistrattionPlatform;
+
+import javax.management.*;
+import javax.management.loading.ClassLoaderRepository;
+import javax.management.remote.JMXConnector;
+import javax.management.remote.JMXConnectorServer;
+import javax.management.remote.JMXConnectorServerFactory;
+import javax.management.remote.JMXServiceURL;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.net.MalformedURLException;
+import java.rmi.registry.LocateRegistry;
+import java.rmi.registry.Registry;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class JMXOshiAgentServer implements JMXOshiAgent {
+
+    private String host;
+    private Integer port;
+    private Map properties;
+
+    private MBeanServer server;
+    private JMXServiceURL address;
+    private JMXConnectorServer cntorServer;
+
+    private ContextRegistrationPlatform contextRegistrationPlatform;
+
+    private static JMXOshiAgentServer jmxOshiAgentServer;
+    private static final PlatformEnum CURRENT_PLATFORM = PlatformEnum.getValue(Platform.getOSType());
```

Comment:
`SystemInfo.getCurrentPlatform()` already gets this, and you only use it once so it's probably fine to use it inline there.

Reply:
got it, and  in the baseboard a made mistake he, yes what i needed was baseboard.getManufacturer()